Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -427,6 +511,14 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
             catch
             {
                 $ServiceFabricSkipped += $ServiceFabric
+                if($AutoRemediation){	
```

Comment:
this conditional statement is not necessary according to me.

Write-Host can be common for standalone script as well, according to me

Reply:
I have removed it.